File: 20216/20216.cpp
#include <iostream>
#include <cstdlib>
#include <string>

using namespace std;

int main(void)
{
	//freopen("input_2.txt", "r", stdin);
	string input;

	while (1) {
		getline(cin, input);
	
		if (input.back() == '?')
			cout << "Quack!" << endl << endl;
		else if (input.back() == '.')
			cout << "*Nod*" << endl << endl;
		else
			exit(0);
	}

	return 0;
}

File: 20216/20216.py
import sys

sys.stdin = open('input_2.txt', 'r')
print()

while True:
	string = input()

	if string == 'I quacked the code!':
		sys.exit()

	if string[-1] == '.':
		print('*Nod*')
	elif string[-1] == '?':
		print('Quack!')

	sys.stdout.flush()

